### ISSUE TITLE
[Accessibility]: Congratulate users on saving a form

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35195]
### Audit type

Screen reader navigation

### User journey audit issue

#33517

### Problem description

<img width="640" alt="Screen Shot 2022-05-09 at 5 29 58 PM" src="https://user-images.githubusercontent.com/103087/167509485-e811ee5e-974e-4c84-82f3-1d0ef12f31be.png">

Saving changes to repository auto-indexing configuration does not announce success. Failure is announced properly.

### Expected behavior

Show a banner on successful save.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by macraig** [comment]
Closing as identified issues have been completed
